1-js/03-code-quality/03-comments/article.md

```diff
@@ -111,70 +111,70 @@ function addJuice(container) {
 }
 ```
 
-Once again, functions themselves tell what's going on. There's nothing to comment. And also the code structure is better when split. It's clear what every function does, what it takes and what it returns.
+एक बार फिर, functions स्वयं बताते हैं कि क्या हो रहा है। टिप्पणी करने के लिए कुछ भी नहीं है। और विभाजित होने पर कोड संरचना भी बेहतर होती है। यह स्पष्ट है कि प्रत्येक function क्या करता है, यह क्या लेता है और यह क्या लौटाता है।
 
-In reality, we can't totally avoid "explanatory" comments. There are complex algorithms. And there are smart "tweaks" for purposes of optimization. But generally we should try to keep the code simple and self-descriptive.
+वास्तव में, हम "व्याख्यात्मक" टिप्पणियों से पूरी तरह से बच नहीं सकते हैं। जटिल एल्गोरिदम होते हैं और अनुकूलन के प्रयोजनों के लिए चतुर "बदलाव" हैं। लेकिन आम तौर पर हमें कोड को सरल और स्व-वर्णनात्मक रखने का प्रयास करना चाहिए।
 
-## Good comments
+## अच्छी टिप्पणियाँ
 
-So, explanatory comments are usually bad. Which comments are good?
+इसलिए, व्याख्यात्मक टिप्पणियां आमतौर पर खराब होती हैं। कौन सी टिप्पणियाँ अच्छी हैं?
 
-Describe the architecture
-: Provide a high-level overview of components, how they interact, what's the control flow in various situations... In short -- the bird's eye view of the code. There's a special language [UML](http://wikipedia.org/wiki/Unified_Modeling_Language) to build high-level architecture diagrams explaining the code. Definitely worth studying.
+वास्तुकला का वर्णन करें
```

Comment:
I think वास्तुकला is not the same architecture we are talking about. Change it to आर्किटेक्चर itself.

Reply:
"संरचना" , इस शब्द का प्रयोग करें?

Reply:
Yeah we can use this as well. Use either आर्किटेक्चर or संरचना (In both line 122 & 123). Make sure we are using same translation for a word throughout the article.